Arduino_related_code/arduinovalvecontrol.txt

```diff
@@ -1,117 +1,210 @@
 // Serial port constants
 const int baud_rate = 9600;
 
-// Control constants 
-const int ONBOARD_LED          = 13;    // digital out to drive the on board LED for heart bit
-const int SLOW_HEART_BIT_DELAY = 500;  // Delay time for slow rate heartbit 
-const int FAST_HEART_BIT_DELAY = 200;  // delay time for the fast rate heartbit
-const int NUM_DIGITAL_CHANNEL  = 16;    // number of digital channels. 
+// Control constants
```

Comment:
If I got it right, the test file is the Arduino FW. You replaced the old with the new one.
However, I included the Arduino-FW folder in the repository.   It's a sub directory in the Arduini_related_code. So the .txt file can be deleted (rather than updated).

What do you think?

Reply:
yes